draft-thomson-train-protocol.md

```diff
@@ -401,14 +401,78 @@ Similarly, if there is a strong need to ensure that a rate limit is respected,
 network elements cannot assume that the signaled limit will be respected by
 endpoints.
 
-## Privacy {#privacy}
+# Privacy considerations {#privacy}
 
-Any network element that can observe the content of that packet can read the
-rate limit that was applied.
-
-{:aside}
-> TODO: Write more.
+In our privacy analysis, we studied whether TRAIN could be used by attackers
+to gain information about the ongoing traffic and compromise the privacy
+of clients or servers by exposing their identity of their activity.
```

Comment:
```suggestion
The focus of this analysis is the extent to which observing TRAIN
packets could be used to gain information about endpoints.
This might be leaking details of how applications using QUIC
operate or leaks of endpoint identity when using additional
privacy protection, such as a VPN.
```